core/src/main/java/org/opentosca/toscana/core/data/Application.java

```diff
@@ -0,0 +1,23 @@
+package org.opentosca.toscana.core.data;
+
+import java.util.Map;
+
+public class Application {
+
+    /**
+     * Stores all scheduled, ongoing or finished transformations of this application.
+     * Key is the platform identifier.
+     */
+    private Map<String, Transformation> transformations;
+    private Csar csar;
+
+    public Application(Csar csar){
+        this.csar = csar;
+    }
+
+    public void transform(Platform targetPlatform){
```

Comment:
If i understood you correctly in the last meetings, this method will invorke the Transformator class to start the transformation asychronously. But how do we get access to the Transformator instance?

Reply:
Almost! This method will create a Transformation and add it to the list of transformations this application holds. This maybe needs to be renamed to createTransformation() 
But the transform() method in Transformation.java will call the Transformator. So, Transformation needs to know Transformator. Concluding, Application has to know Transformator. In the end, it might be the DataAccess class which has to know the transformator in the beginning?

Reply:
opened #40